db/schema.rb

```diff
@@ -49,7 +48,7 @@
     t.string "sector_category"
     t.boolean "ingested", default: false
     t.string "legacy_iati_xml"
-    t.index ["activity_id"], name: "index_activities_on_activity_id"
```

Comment:
Do I need to re add this back?

Reply:
Adding this back in for `parent_id` reduced the page load time from 2 seconds to 1.3 seconds.